README.md

```diff
@@ -84,6 +138,8 @@ Add assets, e.g. images or JavaScript, to [`assets`](assets).
 [`favicon.ico`](favicon.ico) should be served from the root of the project.
 Just replace the current one with the `favicon.ico` of your project.
 
```

Comment:
till here

Reply:
Ok, will remove the Jekyll section, should I wait to hear first from the other reviewers for other feedback before making the change or just go ahead and delete it? Thanks!

Reply:
ya it will be great if we wait for other's review first, so that you can do all the changes in one go